DEPLOY.md

```diff
@@ -55,4 +55,8 @@ $ ./scripts/buildWordlist.ts ../peter-broda-wordlist__scored.txt ../cluedata.txt
 $ GOOGLE_APPLICATION_CREDENTIALS=../serviceAccountKey.json ./scripts/uploadWordlist.ts
 
 # If deploying for prod, now increment version constant in WordDB.ts to force rebuild
-```
\ No newline at end of file
+```
+
+_n.b._ if you're developing on your own Firebase project, you'll need to create Storage before you upload the wordlist: Build > Storage > Get Started > Start in production mode > Save. You don't need to create anything in there, it just needs to be set up before you can upload the wordlist otherwise you'll get a "bucket does not exist" error.
+
+Then you'll need to set read permission on your storage by going to Rules > Edit rules and changing `allow read, write: if false;` to `allow read: if true;`.
```

Comment:
I honestly haven't tested this yet but I think now that we have the `storage.rules` in the repo we should be able to do `$ npx firebase deploy --only storage` to upload the exact rules we're using for crosshare prod. If we can confirm that that works then it might be both easier than messing in the UI and generate a result that's more useful for development.